tests/unit/remote/test_http.py

```diff
@@ -125,3 +125,25 @@ def test_http_method(dvc):
     assert tree._auth_method() == auth
     assert tree.method == "PUT"
     assert isinstance(tree._auth_method(), HTTPBasicAuth)
+
+
+def test_exists(dvc, http, mocker):
+    import io
+
+    import requests
+
+    res = requests.Response()
+    res.raw = io.StringIO("foo")
+    tree = HTTPTree(dvc, http.config)
```

Comment:
We just discussed this in the team, and they pointed out that we could get rid of `dvc` and `http` fixtures as we are already mocking the response.

@damakuno, let's remove `dvc` and `http` from above fixture and let's do the following:
```suggestion
    tree = HTTPTree(None, {})
```

Reply:
I've added a commit with this change